doc/INSTALL

```diff
@@ -246,353 +217,375 @@ General notes for the installation of the 4.2 release of KVIrc
     to "/usr/lib/libgsm.a", you might create the shared library manually
```

Comment:
No idea, I didnt touch this information, but I agree in principle people should either install distro provided libs or compile own libs properly should their distro not provided the prepackaged or specific version, whatever the case may be.

Ill try to make this clear.